CHANGELOG.md

```diff
@@ -19,6 +19,16 @@ cargo-dist has traditionally parsed the version number of new releases and used
 
 * impl @mistydemeo [feat: allow always marking releases as stable](https://github.com/axodotdev/cargo-dist/pull/1054)
 
+## Vendoring external GitHub actions
+
+This feature allows vendoring (almost) all external GitHub actions called by the release workflow. When enabled, specific known versions of all external actions (except `actions/checkout`) are copied into your repository as a part of `cargo dist init`. We don't expect most users to need this feature, but it's provided for users with specific security needs.
```

Comment:
i wonder if there is a way to say this more positively- like, for users that want stronger guarantees, or something. i think saying "we don't expect more users to need this" feels off here

Reply:
Yeah, I was struggling a bit with how to word this. The thing is, since it adds a million lines (not an exaggeration!) of code to the user's repo, I think we want to strike a balance here where we don't come across as *encouraging* it to users who'd be surprised by the weight of it. I think it's more just that people who 100% understand the tradeoffs can make the choice?